webapps-schema/src/main/java/io/camunda/webapps/schema/entities/usermanagement/RoleEntity.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.schema.entities.usermanagement;
+
+import io.camunda.webapps.schema.entities.AbstractExporterEntity;
+import java.util.Set;
+
+public class RoleEntity extends AbstractExporterEntity<RoleEntity> {
+
+  public static final String DEFAULT_TENANT_IDENTIFIER = "<default>";
```

Comment:
I believe this isn't necessary anymore: 
https://github.com/camunda/camunda/blob/main/webapps-schema/src/main/java/io/camunda/webapps/schema/entities/AbstractExporterEntity.java#L20